src/azure-cli/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -388,3 +402,19 @@ def load_command_table(self, _):
                             postgres_private_link_resources_sdk,
                             client_factory=cf_postgres_private_link_resources_operations) as g:
         g.show_command('list', 'list_by_server')
+
+    with self.command_group('mysql server key',
```

Comment:
can we mark the new command group as Preview for one or two sprint to test the behaviors for the commands?